### ISSUE TITLE
Please use CLEAR and SIMPLE language across linux mint and cinnamon

### ISSUE THREAD

**Issue by damnfella** [linuxmint/cinnamon#9071]
### I wanna suggest an #enhancement in the choice of words you use all over the UI

>If the official translation for a given word isn’t commonly known or used by users, you should not use it. Prefer the terms used by most people instead. The goal isn’t to educate users or to enforce anything. Correctness and exactitude are important but not as much as user experience.
[More](https://linuxmint-translation-guide.readthedocs.io/en/latest/verify.html#politics)

The prime reason that I believe why **most people who give Linux Mint a try and then give up** is because the language is not SIMPLE and CLEAR ENOUGH to feel at home. 

- When they move to a new operating system, they should feel right at home. Without making use of "too formal", uncommon or rare words, we can retain the users, and strengthen the spirit of open source. 

For instance, the function of **"Inhibit Applet"**, is not very obvious when I read the name. 
![image](https://user-images.githubusercontent.com/51771172/71561715-1691a880-2aa0-11ea-82a4-4158d91a0a07.png)

**As I continue to use this applet, I see "Power Management inhibited"** 

![image](https://user-images.githubusercontent.com/51771172/71561721-29a47880-2aa0-11ea-84af-665c4e3a5b75.png)

**I also see some misplaced functions:**

![image](https://user-images.githubusercontent.com/51771172/71561741-5eb0cb00-2aa0-11ea-876d-633f89f850a6.png)

***
**Why can't we use SIMPLE language to _DESCRIBE, NAME, and EXPLAIN_ ?**
- Not everyone is fluent enough (or british enough) to know these "high-level" words

Some users do not understand the meaning of words like "inhibit". So they install **extra** apps like caffeine and bloat the system. 

**The user experience is more important than any other consideration.**
Please make the 'language' in the next versions of Cinnamon and Linux Mint easier to understand for all users.

**Comment by brownsr** [comment]
Your point is reasonable, but could you please be specific about the places where you find language that you find hard to understand.  For example where is the 'System Inhibit' that you refer to ?  This does not ring any bells with me, so you need to tell me/us where you are finding it.

Regards

Simon

**Comment by damnfella** [comment]
![image](https://user-images.githubusercontent.com/51771172/71561690-c74b7800-2a9f-11ea-961f-7c873c660e2d.png)

**Comment by brownsr** [comment]
in my opinion that is about as clear as it can be, and I don't see the language of 'System inhibit' and 'Power management inhibited' that you mention.  Are you using the current version  of Cinnamon ?

**Comment by damnfella** [comment]
yes. There are many other places where the language is not simple enough

**Comment by brownsr** [comment]
As I said above you need to be very specific about exactly where these places are, whether you are seeing the language in plan text in the UI, or in tooltips etc.  We need very specific reports to be be able to act on them.

**Comment by damnfella** [comment]
should I open more issues as I find them or drop them here in the comments? Thanks btw.

**Comment by brownsr** [comment]
You may as well put them here, for simplicity.  If you could create one large detailed list in a comment that would be simplest for us to digest.

**Comment by mtwebster** [comment]
Just throwing in my 2 cents, the applet's description explains clearly and in reasonably plain language what this applet does.  It disables notifications and power management.  You've got 2 switches, one for each.

Disabling notifications is not directly related to the other notification applet.  You still get notifications whether that applet is on a panel or not.  Unless you use this switch.

The grouping control of these things in the inhibit applet makes better sense because they're arguably for the same purpose - you don't want to be interrupted, or have distracting popups for some reason - maybe you're doing a presentation or watching a movie.  Most movie players disable power management, but don't necessarily prevent notifications.

The only change I might consider making is the tooltip saying something like "Power management: disabled" rather than inhibited.  Otherwise, I think everything else is fairly plain.

**Comment by JosephMcc** [comment]
I tend to agree with mtwebster and your example isn't very good. You're trying to say that "Inhibit" and the explanation for it isn't clear enough. Then you say that users will install "Caffeine" instead? That certainly isn't more descriptive.

And if you want this done across all of Linux Mint, not just Cinnamon, it's better here: https://github.com/linuxmint/linuxmint/issues

**Comment by damnfella** [comment]
>The grouping control of these things in the inhibit applet makes better sense because they're arguably for the same purpose - you don't want to be interrupted, or have distracting popups for some reason - maybe you're doing a presentation or watching a movie. Most movie players disable power management, but don't necessarily prevent notification.

@mtwebster I'm not concerned about the grouping, sorry about the bad explanation. The notifications applet could use a little "do not disturb" though. Anyway, with that screenshot, I only tried to explain that things are not where they should be.

Don't mind about the examples I gave (inhibit applet), I was being very GENERAL. I didn't actually know where to send it to tell it to you guys. Its not really a technical "issue". 

Also, maybe caffeine sounds like a nice name for disabling power management? I personally don't like "inhibit", even it the description explains it.
